code/modules/mechs/mech_damage.dm

```diff
@@ -197,9 +267,9 @@
 				for(var/thing in pilots)
 					var/mob/pilot = thing
 					pilot.emp_act(severity)
-
+*/
 /mob/living/exosuit/explosion_act(target_power, explosion_handler/handler)
-	var/damage = target_power - getarmor(body, ARMOR_BOMB)
+	var/damage = target_power - (getarmor(body, ARMOR_BOMB) + getarmor(arms, ARMOR_BOMB) + getarmor(legs, ARMOR_BOMB) + getarmor(head, ARMOR_BOMB))/4 // Now uses the average armor of all components
```

Comment:
Might be better to deal damage to each limb separately, so lighter limbs take more explosion damage.